### ISSUE TITLE
DYODE Accessibility/ADA

### ISSUE THREAD

**Issue by jennah619** [jennah619/QADocumentationPractice#33]
**Device name DESKTOP-NM47432-BigMonster
Processor Intel(R) Core(TM) i5-5200U CPU @ 2.20GHz 2.20 GHz
System type 64-bit operating system, x64-based processor
Windows 10**

Chrome-Version 93.0.4577.63 (Official Build) (64-bit)

**Website:** dyode.com

**Description/Summary:**
No ADA link on website

**Expected Behavior:**
ADA is not mandatory for private business, but is considered inclusive and becoming the "norm" for business in general.
A windows screen reader should be able to read entire website.

**Actual Behavior:**
Ran WAVE tool and found 85 Alt text errors. 
Images without alt text cause problems for screen readers because there's no way to communicate the content of the image to the user. Windows Narrator could only read first two paragraphs until it got to the "more clients" page. Narrator then started over and was caught in a loop. 

**Suggest:** Resolve Alt Text errors to make website more accessible to more users.

<img width="960" alt="dyodeWAVEreport" src="https://user-images.githubusercontent.com/79431576/209715695-089509ce-2295-43ae-9c23-ab839acde483.png">
